flytekit/models/core/workflow.py

```diff
@@ -778,13 +801,19 @@ def on_failure(self):
         """
         return self._on_failure
 
+    @property
+    def tags(self):
+        """
+        :rtype: flytekit.models.core.workflow.WorkflowMetadata.Tags
+        """
+        return self._tags
+
     def to_flyte_idl(self):
         """
         :rtype: flyteidl.core.workflow_pb2.WorkflowMetadata
         """
-        workflow_metadata = _core_workflow.WorkflowMetadata()
```

Comment:
`WorkflowMetadata` can't accept optional arguments (see Rust flyteidl, there's no Option<T>).
So I use zero and {} as default constructor.

I think it's better to assign default values in Python side for cases like this. That is, we don't try to modify the function signature in Rust.